gtsfm/two_view_estimator.py

```diff
@@ -26,20 +27,67 @@
 pil_logger.setLevel(logging.INFO)
 
 
+# In case an epipolar geometry can be verified, it is checked whether
+# the geometry describes a planar scene or panoramic view (pure rotation)
+# described by a homography. This is a degenerate case, since epipolar
+# geometry is only defined for a moving camera. If the inlier ratio of
+# a homography comes close to the inlier ratio of the epipolar geometry,
+# a planar or panoramic configuration is assumed.
+# Based on COLMAP's front-end logic here:
+#    https://github.com/colmap/colmap/blob/dev/src/estimators/two_view_geometry.cc#L230
+MAX_H_INLIER_RATIO = 0.8
+
+EPSILON = 1e-6
+
+
+@dataclass(frozen=False)
+class TwoViewEstimationReport:
+    """
+    Args:
+        num_inliers_est_model: #correspondences consistent with estimated model (not necessarily "correct")
```

Comment:
Can we rename to num_inliers and inlier_ratio? 
We could use num_inliers_gt or gt_inlier_ratio for ground truth.

Reply:
marking as resolved from our earlier conversation